scripts/build-linux-deb.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/sh
+
+set -eu
+
+# git repo/branch to use
+GIT_REPO="https://github.com/torvalds/linux"
+GIT_BRANCH="master"
+# base config to use
+CONFIG="defconfig"
+# extra configs to set
+# systemd-boot won't implement support for compressed images (zImage); see
+# https://github.com/systemd/systemd/issues/23788
+EXTRA_CONFIGS="CONFIG_EFI_ZBOOT=y"
```

Comment:
Probably better to separate the extra configs as a config fragment, that way any user can easily customize if needed.

Reply:
Done in a standalone commit to make it easier to review (happy to squash though)
https://github.com/qualcomm-linux/qcom-deb-images/pull/21/commits/6afb8be862332d4a3f823c430df4b32b873a88ab